tests/RXMesh_test/test_attribute.cu

```diff
@@ -138,6 +138,48 @@ TEST(Attribute, Reduce)
 }
 
 
+TEST(Attribute, ArgMax)
+{
+    using namespace rxmesh;
+
+    CUDA_ERROR(cudaDeviceReset());
+
+    RXMeshStatic rx(STRINGIFY(INPUT_DIR) "sphere3.obj");
+
+    auto attr = *rx.add_vertex_attribute<float>("v", 1, rxmesh::DEVICE);
+
+    const float val(2.0);
+
+    populate<float>(rx, *attr, val);
+    auto context = rx.get_context();
+
+    uint32_t chosenVertex = 1;
+    VertexHandle chosenHandle;
+    rx.for_each_vertex(
+        rxmesh::HOST,
+        [attr,context,chosenVertex,&chosenHandle](const rxmesh::VertexHandle vh) 
+        {
+            if (context.linear_id(vh) == chosenVertex) {
+                attr(vh)     = 10;
+                chosenHandle = vh;
+            }
+        },
+        NULL,
+        false);
```

Comment:
This is kinda unnecessary. You can just do `attr(chosenHandle) = 10`

Reply:
The chosen handle isn't initialized and I wasn't sure how to choose a handle. How would I initialize it ?

Reply:
I see. I think this is okay for now